src/debug_menu.cpp

```diff
@@ -121,99 +128,133 @@ class mission_debug
         static std::string describe( const mission &m );
 };
 
-static int player_uilist()
+static bool display_selector( bool display_all_entries, const uilist_entry_pair_t &e )
 {
-    const std::vector<uilist_entry> uilist_initializer = {
-        { DEBUG_MUTATE, true, 'M', _( "Mutate" ) },
-        { DEBUG_CHANGE_SKILLS, true, 's', _( "Change all skills" ) },
-        { DEBUG_LEARN_MA, true, 'l', _( "Learn all melee styles" ) },
-        { DEBUG_UNLOCK_RECIPES, true, 'r', _( "Unlock all recipes" ) },
-        { DEBUG_EDIT_PLAYER, true, 'p', _( "Edit player/NPC" ) },
-        { DEBUG_DAMAGE_SELF, true, 'd', _( "Damage self" ) },
-        { DEBUG_SET_AUTOMOVE, true, 'a', _( "Set automove route" ) }
+    return ( display_all_entries || ( !display_all_entries && e.second ) );
+};
+
+static int player_uilist( bool display_all_entries = true )
+{
+    /* Note: the second boolean indicates whether the menu should always be shown (true) or not (false).
+     * For example, if we show the debug menu on the main menu and we want Info to be always displayed, we set it to true.
+     * This also depends if the top menu is shown or not, in which case,the booleans have no effect if the top menu is not shown.
+     */
+    const std::vector<uilist_entry_pair_t> uilist_initializer = {
+        {uilist_entry( DEBUG_MUTATE, true, 'M', _( "Mutate" ) ), true },
```

Comment:
```suggestion
        { uilist_entry( DEBUG_MUTATE, true, 'M', _( "Mutate" ) ), true },
```
(and similarly for the next six lines).

Reply:
Fixed all similar entries.